Tools/environment_install/install-ROS-ubuntu.sh

```diff
@@ -246,6 +247,26 @@ else
     echo "Skipping adding ardupilot_ws to your home folder."
 fi
 
+
+if maybe_prompt_user "Add ardupilot_gazebo to your home folder [N/y]?" ; then
+    if [ ! -d $AP_GZ_ROOT ]; then
+        git clone https://github.com/khancyr/ardupilot_gazebo
```

Comment:
This should be "https://github.com/ArduPilot/ardupilot_gazebo" I think.

Reply:
Made the changes @rmackay9 sir